XamlControlsGallery/ControlPagesSampleCode/ListView/ListViewSample4_cs.txt

```diff
@@ -0,0 +1,49 @@
+// C# Code
```

Comment:
@stmoy : Why are we not adding those headers per default in the ControlExample? Seems a bit weird having that as a comment in the source code 😅

Reply:
Might not be implemented yet just because not all samples require both a C# and XAML portion to get the point across. Although, I think it would definitely look better if those headers were built in to the ControlExample.